lib/compiler/src/beam_ssa_opt.erl

```diff
@@ -1893,6 +1893,23 @@ reduce_try_is([#b_set{op={succeeded,body}}=I0|Is], Acc) ->
     %% succeeded to the `guard`, since the try/catch will be removed.
     I = I0#b_set{op={succeeded,guard}},
     reduce_try_is(Is, [I|Acc]);
+reduce_try_is([#b_set{op=call,args=[#b_remote{mod=#b_literal{val=M},
+                                              name=#b_literal{val=F},
+                                              arity=A}=R0|Args0]}=I0|Is],
+              Acc) ->
+    %% Rewrite binary_to_atom/1 call to binary_to_atom/2.
+    {I1, Args1} = if {M, F, A} =:= {erlang, binary_to_atom, 1} ->
```

Comment:
Should the same be done for `binary_to_existing_atom`?

Reply:
Yes, thanks for pointing out!